src/libs/ajax/workspaces/providers/ExportWorkflowToWorkspaceProvider.ts

```diff
@@ -40,3 +40,33 @@ export const makeExportWorkflowFromWorkspaceProvider = (
     },
   };
 };
+
+/**
+ * Create a provider to export a workflow from the Broad Methods Repository to a
+ * workspace, with a blank configuration.
+ *
+ * @param {MethodRepoMethod} sourceMethod - the method to be exported from the
+ * Methods Repository. The methodNamespace, methodName, and methodVersion
+ * properties should be present.
+ */
+export const makeExportWorkflowFromMethodsRepoProvider = (
+  sourceMethod: MethodRepoMethod
+): ExportWorkflowToWorkspaceProvider => {
+  return {
+    export: async (destWorkspace: WorkspaceInfo, destWorkflowName: string, options: AbortOption = {}) => {
+      const { signal } = options;
+
+      // Remove placeholder root entity type from template before importing -
+      // the user can select their own on the workflow configuration page
+      const { rootEntityType, ...template } = await Ajax(signal).Methods.template(sourceMethod);
+
+      return Ajax(signal)
+        .Workspaces.workspace(destWorkspace.namespace, destWorkspace.name)
```

Comment:
Are you referring to adding a type to the parameter of `importMethodConfig` in `src/libs/ajax/workspaces/Workspaces.ts`? (I want to check since this Ajax function was preexisting.)

Reply:
Yes. I just took a pass through the Workspaces ajax file and added/moved types for everything that I could. I wasn't able to do much for workflows methods, but since you know the expected type of the input argument this would be a good time to add it.

Reply:
Updated!